src/template.html

```diff
@@ -29,5 +28,7 @@
 	<!-- The application will be rendered inside this element,
 	     because `src/client.js` references it -->
 	<div id="sapper">%sapper.html%</div>
+
```

Comment:
can you revert the changes to this file? no need for the blank lines at the end and the comment no longer aligns